ee/clickhouse/queries/event_query.py

```diff
@@ -103,11 +104,6 @@ def _determine_should_join_persons(self) -> None:
             self._should_join_persons = True
             return
 
-        if self._filter.breakdown_type == "person":
```

Comment:
Q: Why remove this? Was this handled by column_optimizer already?

Reply:
no idea i think it got deleted when i merged in your branch because of previously edited deletions